pkg/client/example-runner_test.go

```diff
@@ -17,6 +21,79 @@ func ExampleGetAllCompleteServiceExamples() {
 	// <nil>
 }
 
+func ExampleGetAllCompleteServiceExamples2() {
```

Comment:
If you want more than one example against a given type, use an underscore then a camelCase description heading to make it show up in godoc e.g. `ExampleGetAllCompleteServiceExamples_jsonSpec`